slasher/main.go

```diff
@@ -26,13 +26,25 @@ func startSlasher(ctx *cli.Context) error {
 		return err
 	}
 	logrus.SetLevel(level)
-	port := ctx.GlobalString(flags.RPCPort.Name)
+	port := ctx.GlobalInt(flags.RPCPort.Name)
 	cert := ctx.GlobalString(flags.CertFlag.Name)
 	key := ctx.GlobalString(flags.KeyFlag.Name)
+	beaconCert := ctx.GlobalString(flags.BeaconCert.Name)
+	beaconPort := ctx.GlobalInt(flags.BeaconPort.Name)
+	beaconProvider := ctx.GlobalString(flags.BeaconProvider.Name)
```

Comment:
can you make the beacon provider and port come under one flag ?

Reply:
for sure